tests/tasks/assert_certificate_parameters.yml

```
@@ -76,13 +78,14 @@
       {{ cert['group'] | default('root') }}
 
 - name: Parse certificate
-  command: "{{ __virtualenv_path }}/bin/certreader2json {{ cert['path'] }}"
-  register: certreader_result
+  certtojson:
+    filename: "{{ cert['path'] }}"
+  register: certificate_data
   changed_when: false
```

```suggestion
  changed_when: false
  vars:
    ansible_python_interpreter: "{{ __virtualenv_path }}/bin/python"
```
that's how you make ansible use the python from the venv
```

---

Thank you. I'm testing locally before pushing the change. It is taking some time because connection is pretty slow.